ticker/src/main/java/com/robinhood/ticker/TickerColumn.java

```diff
@@ -209,6 +212,12 @@ private boolean drawText(Canvas canvas, Paint textPaint, char[] characterList, i
         if (index >= 0 && index < characterList.length) {
             canvas.drawText(characterList, index, 1, 0f, verticalOffset, textPaint);
             return true;
+        } else if (startIndex == UNKNOWN_START_INDEX && index == UNKNOWN_START_INDEX) {
```

Comment:
just want to make sure: do these cases handle animating from an unknown char to and unknown char? and what does that look like:

[known chars][unknown char 1][unknown char 2]?